gotify-icon-updater/README.md

```diff
@@ -0,0 +1,95 @@
+# gotify-icon-updater
```

Comment:
Hey, the gotify/contrib is _not_ for source code. It is only a list of links to other projects. Please add your tool/script to a separate repository, blog post, gist, etc..